### DISCUSSION TITLE
Checking dockers recourse use

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#8131] [Unsupported Versions] [answered]
Hello all,

SO 2.3.130  stanalone iso install
suricata meta
8 vCPU
24 vRAM
1.5 TB diskspace

Is there a way to see how mutch recources of the host a docker uses? like cpu & ram usage. 

This is because SO stated yesterday  98% mean cpu usage in grafana and normal it is 64%. I would like to know what caused it if this happens again.

Regards
Bart

**Comment by weslambert** [comment, marked as answer]
Grafana should give you a pretty good understanding of resource utilization. You could also run `docker stats` on the host to get an idea for each container.

**Reply by sleepingbel** [reply]
thx Wes the docker stats command was what i needed.
